internal/services/mobilenetwork/mobile_network_packet_core_data_plane_resource.go

```diff
@@ -0,0 +1,286 @@
+package mobilenetwork
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/mobilenetwork/2022-11-01/packetcorecontrolplane"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/mobilenetwork/2022-11-01/packetcoredataplane"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type PacketCoreDataPlaneModel struct {
+	Name                                  string            `tfschema:"name"`
+	MobileNetworkPacketCoreControlPlaneId string            `tfschema:"mobile_network_packet_core_control_plane_id"`
+	Location                              string            `tfschema:"location"`
+	Tags                                  map[string]string `tfschema:"tags"`
+	UserPlaneAccessIPv4Address            string            `tfschema:"user_plane_access_ipv4_address"`
+	UserPlaneAccessIPv4Gateway            string            `tfschema:"user_plane_access_ipv4_gateway"`
+	UserPlaneAccessIPv4Subnet             string            `tfschema:"user_plane_access_ipv4_subnet"`
+	UserPlaneAccessName                   string            `tfschema:"user_plane_access_name"`
```

Comment:
is there a reason these are not in a block `user_plane_access` like the api?

Reply:
Because the service is always returning it, while we do not want to let it be in a `computed` block. like the comment in another PR https://github.com/hashicorp/terraform-provider-azurerm/pull/20730#discussion_r1143185785

Reply:
👍